packages/swc-plugin-obsidian/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "swc-plugin-obsidian",
+  "version": "3.0.0-alpha.6",
+  "description": "Dependency injection framework for React and React Native applications",
+  "scripts": {
+    "prepack": "npm run lint && tsc --project tsconfig.prod.json",
+    "lint": "eslint",
+    "build": "tsc --project tsconfig.json",
+    "pretest": "tsc --project tsconfig.json",
+    "test": "npx jest",
+    "test-coverage": "npx jest --coverage"
+  },
+  "main": "dist/src/index.js",
+  "directories": {
+    "test": "test"
+  },
+  "devDependencies": {
+    "@babel/preset-typescript": "^7.26.0",
+    "@eslint/compat": "^1.1.1",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.9.1",
+    "@stylistic/eslint-plugin": "^2.7.2",
+    "@swc/core": "^1.7.39",
+    "@swc/jest": "^0.2.36",
+    "@types/jest": "^29.5.13",
+    "@types/jest-when": "^3.5.5",
+    "@typescript-eslint/eslint-plugin": "^8.4.0",
+    "@typescript-eslint/parser": "^8.4.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^9.9.1",
+    "eslint-plugin-jest": "^28.8.3",
+    "eslint-plugin-unused-imports": "^4.1.3",
+    "globals": "^15.9.0",
+    "jest": "29.7.x",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-extended": "^4.0.0",
```

Comment:
Some of these dependencies aren't used - we should clean up the dev dependencies later. I'm pretty sure `jest-extended` and `jest-when` aren't used.